.gitmodules

```diff
@@ -1,7 +1,7 @@
 [submodule "atmos_cubed_sphere"]
 	path = atmos_cubed_sphere
-	url = https://github.com/NOAA-EMC/GFDL_atmos_cubed_sphere
-	branch = dev/emc
+	url = https://github.com/XiaqiongZhou-NOAA/GFDL_atmos_cubed_sphere
+	branch = pd-5
 [submodule "ccpp/framework"]
 	path = ccpp/framework
 	url = https://github.com/NCAR/ccpp-framework
```

Comment:
sorry, I may miss this, what is the ccpp framework change for?

Reply:
This is correct and we discussed it beforehand. The change simply sorts the list of files for cmake to compile to avoid unnecessary recompiling if files haven't changed. We merged the ccpp-framework PR after you and Dusan agreed to do the change with Kate's dycore update. It obviously doesn't change the results or anything except speeding up recompiling.